crest/Assets/Crest/Crest/Scripts/Shapes/FFT/ShapeFFT.cs

```diff
@@ -437,6 +457,12 @@ public void OnSplinePointDrawGizmosSelected(SplinePoint point)
         {
             DrawMesh();
         }
+
+        // Called by Predicated attribute. Signature must not be changed.
+        bool IsLocalWaves(Component component)
```

Comment:
I'm proposing to add a Mode setting in painting pr which can be used here

Reply:
Sounds good. I'll wait until that merges.

Reply:
I'd say whichever comes first, not sure how close painting is so perhaps could be done later if this merges first

Reply:
Done.